pkg/stanza/fileconsumer/attrs/attrs.go

```diff
@@ -5,25 +5,57 @@ package attrs // import "github.com/open-telemetry/opentelemetry-collector-contr
 
 import (
 	"fmt"
+	"os"
+	"os/user"
 	"path/filepath"
 	"runtime"
+	"syscall"
 )
 
 const (
 	LogFileName         = "log.file.name"
 	LogFilePath         = "log.file.path"
 	LogFileNameResolved = "log.file.name_resolved"
 	LogFilePathResolved = "log.file.path_resolved"
+	LogFileOwnerName    = "log.file.owner.name"
+	LogFileGroupName    = "log.file.group.name"
 )
 
 type Resolver struct {
 	IncludeFileName         bool `mapstructure:"include_file_name,omitempty"`
 	IncludeFilePath         bool `mapstructure:"include_file_path,omitempty"`
 	IncludeFileNameResolved bool `mapstructure:"include_file_name_resolved,omitempty"`
 	IncludeFilePathResolved bool `mapstructure:"include_file_path_resolved,omitempty"`
+	IncludeFileOwnerName    bool `mapstructure:"include_file_owner_name,omitempty"`
+	IncludeFileGroupName    bool `mapstructure:"include_file_group_name,omitempty"`
 }
 
-func (r *Resolver) Resolve(path string) (attributes map[string]any, err error) {
+func (r *Resolver) AddFileInfos(file *os.File, attributes map[string]any) (err error) {
```

Comment:
Since this is just a helper function, we don't need to export it. Maybe call it `addOwnerInfo`?